### ISSUE TITLE
VALIDATIONCLOCK format

### ISSUE THREAD

**Issue by martin-lindstrom** [eu-digital-green-certificates/dgc-testdata#35]
When I parse the VALIDATIONCLOCK values supplied in all common files (and also AT) I get errors saying that it is not correct ISO-8601.

**Comment by martin-lindstrom** [comment]
@SchulzeStTSI That would mean that Sweden's validation code would be 100% correct ;-) But step one is to have correct test files.

Check https://github.com/DIGGSweden/dgc-java and "borrow" what you like. 

Btw: Many of the testfiles under commons are fawlty (missing required fields, string instead of JSON-object, etc.)

**Comment by nodh** [comment]
@martin-lindstrom I'm using [ISO_DATE_TIME](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_DATE_TIME), in good faith that it would produce correct serializations.  I've also had a look at [RFC3339](https://datatracker.ietf.org/doc/html/rfc3339), and to me it seems that `2021-05-03T20:00:00+02:00` is valid example of an ISO-8601 date/time string. But what's wrong exactly?

**Comment by nodh** [comment]
Agree, Zulu Time should be used. Is there anything else to improve on?

**Comment by martin-lindstrom** [comment]
This was very good. A typical thing that should be discovered during interop-testing. People SHOULD use Zulu time, but validators MUST be able to handle offset versions as well.

**Comment by yspreen** [comment]
This is what we support in swift. Feel free to open a PR in both the JSONSchema Swift validator and our core repo to change this behavior.

```
yyyy-MM-dd'T'HH:mm:ssZZZZZ // or
yyyy-MM-dd'T'HH:mm:ss.SSSZZZZZ // or
yyyy-MM-dd't'HH:mm:ss.SSS'z'
```

**Comment by SchulzeStTSI** [comment]
@martin-lindstrom @nodh So far i see we should stick on the 2021-05-03T22:00:00Z format. The Offset can make some trouble in some implementations. What do you think? I looks for me more straight forward regarding the JSONs and Schemas.
